src/ott/geometry/costs.py

```diff
@@ -280,6 +280,45 @@ def twist_operator(
       return vec + jax.grad(self.h_legendre)(-dual_vec)
     return vec - jax.grad(self.h_legendre)(dual_vec)
 
+  def transport_map(self,
+                    f: Func) -> Callable[[jnp.ndarray, bool, Any], jnp.ndarray]:
+    r"""Get an optimal transport map for a concave function :math:`f`.
+
+    Uses Theorem 1.17 from :cite:`santambrogio:15` to define an OT map, e.g. in
```

Comment:
I would probably refer to our paper, I think it provides more context to do what's done in this section.